redCOMPONENT-COM/redSHOP
# REDSHOP-2253 Adding new tag {product_price_percent_discount} to show discount in percentage

#### Testing info
- Add {product_price_percent_discount} in product and cart template. 
- Make sure you get correct discount in percentage.
